### PR TITLE
adds note to blueprint regarding lazy consensus option

### PR DESCRIPTION
**Pull request by mitchell852**
This PR adds a note to the blueprint section of CONTRIBUTING.md regarding the option to invoke [lazy consensus](https://community.apache.org/committers/lazyConsensus.html) on blueprints if needed.

## Which Traffic Control components are affected by this PR?

- Documentation

## What is the best way to verify this PR?
Read Contributing.md file, blueprint section

## PR submission checklist
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: CONTRIBUTING.md
Comment (ocket8888) [inline CONTRIBUTING.md:71]: Why are you using an HTML anchor tag instead of just a markdown link?

**Review by ocket8888 — APPROVED** [review]
Seems fine to me

File: CONTRIBUTING.md
Comment (zrhoffman) [inline CONTRIBUTING.md:71]: Maybe mention that lazy consensus cannot be invoked once any feedback is left.

**Review by zrhoffman — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md
Comment (mitchell852) [reply CONTRIBUTING.md:71]: isn't that sort of implied with

> or feedback has stalled

> you intend to invoke lazy consensus after X hours with no feedback

Comment (zrhoffman) [reply CONTRIBUTING.md:71]: "feedback has stalled" implies that some feedback has already been left. Is there a precedent for lazy consensus being used, in this case?

Comment (rawlinp) [reply CONTRIBUTING.md:71]: I would leave that up to the discretion of the blueprint submitter IMO. Let's say someone reviews your blueprint then goes on vacation for two weeks. Depending on their feedback, you might think they'd be ok with the blueprint merging once their feedback is addressed, rather than forcing you to wait 2 weeks for them to get back. Plus, it's not like once a blueprint is reviewed and merged that the design cannot change after that. Merging a blueprint just generally means you should be able to move forward with the proposed design, although that's not a hard rule either. In the above scenario, if the person who left feedback still didn't agree with the blueprint after it was merged, the blueprint could always be changed (assuming general consensus on the new change).

Comment (zrhoffman) [reply CONTRIBUTING.md:71]: Sounds good to me

**Review by zrhoffman — APPROVED** [review]

**Pull request merged**
